README.md

```diff
@@ -59,7 +59,7 @@ The sender DataApp should send a request using the following schema, specifing i
 ```
 curl -k -request POST 'https://{IPADDRESS}:{A_ENDPOINT_PUBLIC_PORT}/incoming-data-app/multipartMessageBodyBinary' 
    --header 'Content-Type: multipart/mixed; boundary=CQWZRdCCXr5aIuonjmRXF-QzcZ2Kyi4Dkn6' 
-   --header 'Forward-To: {RECEIVER_IP_ADDRESS}:{B_ENDPOINT_PUBLIC_PORT}/incoming-data-channel/receivedMessage' 
+   --header 'Forward-To: {RECEIVER_IP_ADDRESS}:{B_ENDPOINT_PUBLIC_PORT}/data' 
```

Comment:
Good catch to update README file in general 👍

Reply:
Thanks 👍